src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftPermissionFilter.java

```diff
@@ -186,7 +186,9 @@ public void doFilter(ServletRequest request, ServletResponse response,
                                 if (entry == null) {
                                     entry = new BearerCacheEntry();
                                     bearerCache.put(token, entry);
-                                    entry.lastCheck = 0;
+                                    // make last check long.MAX so that the first time
+                                    // this user/token is used, we force the flow
+                                    entry.lastCheck = Long.MAX_VALUE;
```

Comment:
isn't this going to make it never go through the flow because:
`System.currentTimeMillis()- entry.lastCheck > (interval * 1000)` will always be true?

Reply:
sorry, I guess it will always be false now.  Can you elaborate on what this actually fixes though, and why we haven't(?) seen it elsewhere?